aptos-move/e2e-move-tests/src/aggregator.rs

```diff
@@ -7,8 +7,7 @@ use language_e2e_tests::account::Account;
 use std::path::PathBuf;
 
 pub fn initialize(path: PathBuf) -> (MoveHarness, Account) {
-    // TODO: when parallel execution supports deltas - change to `new_mainnet`.
-    let mut harness = MoveHarness::new_no_parallel();
+    let mut harness = MoveHarness::new_mainnet();
```

Comment:
i know i asked before, just to confirm, would this test both sequential and parallel? we should make sure we do that

Reply:
It would run both and compare the outputs, see in https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/e2e-tests/src/executor.rs#L374